PluginGemfile

```diff
@@ -3,9 +3,9 @@ source 'https://rubygems.org'
 gem 'view_component'
 
 group :development, :test do
-  gem 'bullet'
-  gem 'factory_bot_rails'
+  gem 'bullet', '~> 6.1'
+  gem 'factory_bot_rails', '~> 6.4'
   gem 'rubocop', '~> 1.26.0'
   gem 'rubocop-performance', '~> 1.13.0'
-  gem 'brakeman'
+  gem 'brakeman', '~> 6.0'
```

Comment:
Why do you want to lock them so tightly?

Reply:
that is not part of this PR, it's just a merge